users/users-router.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const router = express.Router()
+
+const bcrypt = require('bcryptjs');
+
+const db = require('../data/dbConfig.js');
+
+router.post('/register', (req, res) => {
+    let user = req.body;
+    const hash = bcrypt.hashSync(user.password, 8)
+    user.password = hash;
+
+   
+    db('users')
+    .insert(user)
+    .then(ids => {
+        const id = ids[0];
+        db('users')
+        .where({id})
+        .then(users => {
+            res.status(201).json(users)
+        })
+    })
```

Comment:
I'm still not quite sure why the db needs to be called here twice - I've always done it once and it works just fine. If this makes more sense to you and you can justify it, however, go for it.